app/controllers/tweets_controller.rb

```diff
@@ -0,0 +1,102 @@
+class TweetsController < ApplicationController
+
+  configure do
+    enable :sessions
+    set :session_secret, "secret"
+  end
+
+  get '/tweets' do
+    if logged_in?
+      @users = current_user
+      @tweets = Tweet.all
+      erb :'/tweets/tweets'
+    else
+      redirect to "/login"
+    end
+
+  end
+
+
+  get '/tweets/new' do
+    if logged_in?
+      @user = current_user
+      erb :'tweets/create_tweet'
+    else
+      redirect "/login"
+    end
+  end
+
+  post '/tweets' do
+    @tweet = Tweet.new(params)
+    @user = current_user
+    #check if tweet is empty.
+    # This is where we set the name for song/ it want us to pass in an hash.
+    if logged_in? && !@tweet.content.blank? && @tweet.save
+      @user.tweets << @tweet
```

Comment:
The tweet instantiation and user association can both be accomplished in the same step:

``` ruby
@tweet = Tweet.new(content: params[:content], user: current_user)
```